docs/kbs_attestation_protocol.md

```diff
@@ -0,0 +1,282 @@
+            "symkey-length": "",
+            /* The symmetric key used to encrypt `api-output`, which is encrypted by HW-TEE's public key */
+            "enc-symkey": ""
+	    },
+	    /* The token issued by KBS to KBC. If there is a token in the request, this field will be left blank. */
+	    "token": "",
+	}
+}
+```
+
+- `output`
+
+The output of KBS service API, it needs to be encrypted by a symmetric key randomly generated by KBS. 
+
+- `crypto-annotation`
+
+The symmetric key and algorithm used to encrypt `api-output`. Because the output of KBS service 
+API may contain large data, it is inefficient to directly use HW-TEE's public key for encryption. 
+Therefore, digital envelope technology is used here, KBS generate a random symmetric key to 
+encrypt `api-output`, and then use HW-TEE's public key to encrypt the random symmetric key.
+
+- `token`
+
+If the attestation evidence sent by KBC is verified before returning the response, KBS will issue 
+a token to KBC in this field and sign it with KBS's root key. KBS will maintain the corresponding 
+relationship between each issued token and its original attestation result. KBC can use this token 
+to request resources or services from KBS without Challenge and Attestation within the validity period of 
+the token. If there is already a token in the KBC's request, no new token will be issued and this 
+field will be left blank.
+
+The [JSON web token](https://jwt.io/) standard is adopted for the token. The format is as follows:
+
+(Note that the `/*...*/` comments are not valid in JSON web token, and must not be used in real token.)
+
+```json
+{
+    /* JWT header */
+    "alg": "HS256",
+    "typ": "JWT"
+}.
+{
+    /* JWT Payload official claim set */
+  
+    /* Token's expiration time */
+    "exp": 1568187398,
+    /* Token's issuing time */
+    "iat": 1568158598,
+    /* Token's issuer, which is the URL address of KBS */
+    "iss": "https://xxxxxx",
+    
+    /* JWT payload custom claim set */
+  
+    /* The HW-TEE's public key sent by KBC in the attestation evidence, 
+       which is valid within the validity period of the token. */
+    "tee-pubkey": {
+	    "algo": "",
+	    "pubkey-length": "",
+	    "pubkey": ""
+	}
+}.
+[Signature]
+```
+
+The header part of the token declares the format standard of the token and the signature algorithm used.
+
+The payload part is divided into official claim set and custom claim set. In the official claim 
+set, we select three fields, `exp`, `iat`' and `iss`, which respectively declare the expiration 
+time, issuing time and issuer (KBS URL address) of the token. The custom field in the payload 
+contains the HW-TEE's public key sent by KBC along with the attestation evidence, which is valid 
+within the validity period of the token. When KBC uses the token to request resources or services 
+of KBS service API, KBS uses this public key recorded in the token to protect `api-output` with 
+digital envelope technology.
```

Comment:
Isn't this public key used to encrypt the symmetric key used to encrypt the api-output?

Above line implies that the the public key is used to encrypt the api-output.